main.tf

```diff
@@ -19,20 +19,22 @@
  *****************************************/
 locals {}
 
-resource "google_bigquery_dataset" "default" {
-  dataset_id                  = "${var.dataset_id}"
-  friendly_name               = "${var.dataset_name}"
-  description                 = "${var.description}"
+resource "google_bigquery_dataset" "main" {
+  dataset_id    = "${var.dataset_id}"
+  friendly_name = "${var.dataset_name}"
+  description   = "${var.description}"
+
```

Comment:
We omitted an "access" block for the dataset (https://www.terraform.io/docs/providers/google/r/bigquery_dataset.html#access). While optional, would it make sense to put some default values in to show how to leverage it?